src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/NetHttpsBindingTests.cs

```diff
@@ -5,15 +5,13 @@
 
 using System;
 using System.ServiceModel;
-using System.ServiceModel.Channels;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
+
+using Infrastructure.Common;
 using Xunit;
 
-public static class Binding_Http_NetHttpsBindingTests
+public class Binding_Http_NetHttpsBindingTests: ConditionalWcfTest
 {
-    [Fact]
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
```

Comment:
I will keep these tests consistent with other tests that need Cert installed. We will have a general way to solve the issue that they will not run for NET Native.